### PR TITLE
Registryui

### PR DESCRIPTION
**Pull request by ptdatta**
**Notes for Reviewers**

This PR fixes 
1. Rounded and Square corners. ✅
2. Long vertical line. ✅
3. Remove animation. ✅
4. Each branch of the tree lazily loaded. ✅
5. Eliminate the need for the scrollbar - restyle it when it does need to be displayed.✅
6. Expand all / Collapse all to simply be standard icons. ✅
7. Eleminate X✅
8. Search doesn't work. ✅

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — CHANGES_REQUESTED** [review]
Detailed feedback provided in Slack.

File: ui/utils/data-table.js [hunk 1]
```diff
@@ -100,6 +100,7 @@ const ResponsiveDataTable = ({
       }
     });
     updateCols([...columns]);
+    console.log(data);
```
Comment (sudhanshutech) [inline ui/utils/data-table.js:103]: remove this please

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by sudhanshutech** [comment]
at certain width when the search expands hide the collapse icons and switcher and again visible as the search closes
image

**Comment by sudhanshutech** [comment]
remove the count from tabs after certain width you will get some extra space
image

**Comment by sudhanshutech** [comment]
Make the tabs horizonal scroll when they start squezing

**Comment by sudhanshutech** [comment]
Have you thought any thing for this @ptdatta ?    // @ritiksaxena124 

image

**Comment by sudhanshutech** [comment]
@ritiksaxena124 after certain width can be remove overview tab and instead display a back arrow icon ? // @ptdatta

**Comment by ptdatta** [comment]
@sudhanshutech actually there is another PR which contains the registry mobile view which not yet merged.

**Comment by leecalcote** [comment]
```
#28 216.8 Module not found: Can't resolve 'styled-jsx/server' in '/ui/pages'
```

**Comment by ritiksaxena124** [comment]
> @ritiksaxena124 after certain width can be remove overview tab and instead display a back arrow icon ? // @ptdatta

We are completely removing the Overview tab. It will only be shown when the user clicks the `Registry` tab.

**Comment by leecalcote** [comment]
Resyncing fork.

**Comment by leecalcote** [comment]
While the Registry may still have bugs that @ptdatta _will immediately fix_, we might be able to accept the current set of fixes so that existing users can stop suffering with the current implementation in the meantime. @ptdatta _hurry up_.

**Comment by leecalcote** [comment]
@ptdatta done?

**Comment by ptdatta** [comment]
> @ptdatta done?

Yes✅

**Pull request merged**